utility/src/main/java/org/oppia/android/util/networking/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+"""
+Utilities corresponding to networking utilities.
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "network_conecction_util",
```

Comment:
Typo here: should be ``network_connection_util``. Please make sure to double check spelling before sending for review.